pages/cloud/nutanix/23-ovh-gateway-doc/guide.de-de.md

```diff
@@ -362,10 +536,10 @@ Create the `vm.json` file:
 ```
 
 > [!primary]
-> You can adjust the settings with your values as needed: VM name, number of VCPUs, RAM size, disk size etc.
+> You can adjust the parameters according to your values, according to your needs: VM name, number of VCPU, RAM size, disk size, etc.
```

Comment:
Rewording:
"You can adjust the parameters with your values, according to your needs:"